contrib/FreeBSD/sopel-default.cfg

```diff
@@ -0,0 +1,26 @@
+#
```

Comment:
Would it be worth suggesting use of `sopel configure`, either through the service manager or directly?

Reply:
The service file has a wrapper to `sopel configure`. I think using a wrapper like `service sopel configure` is more convenient because constantly the user is running `service sopel <start|stop|status|restart|configure>`.

Reply:
Right, if that's available, it would be good to mention it - either in this file or in the readme.

Reply:
I will suggest that in the README